core/base-service/deprecated-service.js

```diff
@@ -2,20 +2,19 @@ import Joi from 'joi'
 import camelcase from 'camelcase'
 import BaseService from './base.js'
 import { isValidCategory } from './categories.js'
-import { Deprecated } from './errors.js'
 import { isValidRoute } from './route.js'
 
 const attrSchema = Joi.object({
   route: isValidRoute,
   name: Joi.string(),
   label: Joi.string(),
   category: isValidCategory,
-  message: Joi.string(),
```

Comment:
This was not used by any of the existing callers of `deprecatedService`. I preferred introducing a separate `issueUrl` parameter that sets all of `message`, `link`, and `color`, to ensure we don't forget one or the other when deprecating redirectors in the future.